src/docs/reference/modules/Recipes/README.md

```diff
@@ -107,7 +107,8 @@ The Content step allows you to import some content types.
 
 ### Lucene Step
 
-The Lucene index step allows you to run the Lucene indexation of content types.
+The Lucene index step allows you to run the Lucene indexation of content types.  
+You can also set the default Lucene Settings.
```

Comment:
That's the default Lucene "Search" Settings. We should probably rename that config but probably better not.

Reply:
Ok nevermind keep it like that